### ISSUE TITLE
Hard to see

### ISSUE THREAD

**Issue by lrwyatt** [OHSU-Library/github-tutorial#10]
From the back of the room, seeing Johnny 5 was tough. Might be helpful to have an aide circulating to point things out/answer questions.

**Comment by jmcmurry** [comment]
Good point; I was thinking that it would have been good to pair more seasoned users with those just starting out. We also could have had all participants join the zoom so that they could see Dan's screen. Not sure the internet would have been up to the task though.

**Comment by lrwyatt** [comment]
Julie,

Yeah, the internet/zoom was being a bit disappointing. Pairing up would be good too. I wonder if reconfiguring the tables and chairs could help with that too.

Also, just curious, I only see 3 members (Marijane, Shahim, and Dan) for the OHSU Library on GitHub. Do you know why that might be?

-Letisha

**Comment by jmcmurry** [comment]
Not all members are public; that is configurable by the individual person. If you're logged in though, you *should* see even the private members.

**Comment by lrwyatt** [comment]
Weird. I am signed in but still see 3.

**Comment by jmcmurry** [comment]
The people you're seeing are definitely the public ones, so I must be mistaken. Only admins can see.

**Comment by WorexSDK** [comment]
src/Badges/Badges.test.sol
:&₺-(-_!(?
